kong/init.lua

```diff
@@ -293,9 +307,20 @@ end
 local function execute_plugins_iterator(plugins_iterator, phase, ctx)
   local old_ws = ctx.workspace
   for plugin, configuration in plugins_iterator:iterate(phase, ctx) do
+    local span
+    if phase == "rewrite" then
+      span = instrumentation.plugin_rewrite(plugin)
+    elseif phase == "header_filter" then
+      span = instrumentation.plugin_header_filter(plugin)
+    end
```

Comment:
Could we just do this?
```lua
instrumentation.plugin_phase(phase, plugin)
```

(exposing `plugin_callback` with name `plugin_phase`)

Reply:
The function itself could be no-op, so I think use `plugin_xxx` format could be more easier.